jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ui/components/DiffPatchDialog.kt

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.ui.components
+
+import com.intellij.openapi.diff.impl.patch.FilePatch
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vcs.changes.Change
+import com.intellij.openapi.vcs.changes.ChangeList
+import com.intellij.openapi.vcs.changes.patch.ApplyPatchDifferentiatedDialog
+import com.intellij.openapi.vcs.changes.patch.ApplyPatchExecutor
+import com.intellij.openapi.vcs.changes.patch.ApplyPatchMode
+import com.intellij.openapi.vcs.changes.shelf.ShelvedBinaryFilePatch
+import com.intellij.openapi.vfs.VirtualFile
+import software.aws.toolkits.jetbrains.services.codemodernizer.model.JobId
+import software.aws.toolkits.jetbrains.services.codemodernizer.state.CodeTransformTelemetryState
+import software.aws.toolkits.telemetry.CodeTransformPatchViewerCancelSrcComponents
+import software.aws.toolkits.telemetry.CodetransformTelemetry
+
+class DiffPatchDialog(
```

Comment:
Awesome stuff!

1. We have tested the events emit?
2. We have tested the UI works as expected for the diff? Can you post a screenshot or screenshare in the conversation section for others to visualize the change works.

Reply:
Yes I did! I'll try to post a small recording

Reply:
I have added a screen recording